src/Integration/Language.cs

```diff
@@ -1,22 +1,31 @@
-﻿using SonarLint.VisualStudio.Integration.Resources;
+﻿//-----------------------------------------------------------------------
+// <copyright file="Language.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using SonarLint.VisualStudio.Integration.Resources;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Diagnostics;
 using System.Linq;
 
 namespace SonarLint.VisualStudio.Integration
 {
-    [DebuggerDisplay("{Name} (ServerKey: {ServerKey}, ProjectType: {ProjectType}, IsSupported: {IsSupported})")]
+    [DebuggerDisplay("{Name} (ID: {Id}, ProjectType: {ProjectType}, IsSupported: {IsSupported})")]
+    [TypeConverter(typeof(LanguageConverter))]
     public sealed class Language : IEquatable<Language>
```

Comment:
To be usable as a collection key, one must implement:

- [x] `bool object.Equals`,
- [x] `int object.GetHashCode`,
- [x] `IEquatable<T>` (optional)